genome_designer/genome_finish/contig_display_utils.py

```diff
@@ -1,17 +1,30 @@
 from collections import namedtuple
 
+import settings
+
+
 Junction = namedtuple('Junction',
         ['ref', 'ref_count', 'contig', 'contig_count'])
 
 
 def get_ref_jbrowse_link(contig, loc):
-        return (contig.parent_reference_genome.get_client_jbrowse_link() +
-                '&loc=' + str(loc))
+    sample_alignment = contig.experiment_sample_to_alignment
+    bam_dataset = sample_alignment.dataset_set.get(
+            type='BWA BAM')
```

Comment:
Yeah this is not ideal, but not big enough issue to block. Can you basically just add your note above as a comment in the code as well as a `# TODO(gleb): Figure out better organization`.

You could solve this by redefining those strings as constants in a third file named `constants.py` or something and have both models.py and contig_display_utils.py import from it.